pkg/translator/zipkin/zipkinv1/status_code.go

```diff
@@ -80,15 +82,18 @@ func (m *statusMapper) ocStatus() *tracepb.Status {
 		if s.codePtr != nil {
 			code = *s.codePtr
 		}
-		return &tracepb.Status{
-			Code:    code,
-			Message: s.message,
+		switch code {
+		case trace.StatusCodeOK:
+			dest.SetCode(ptrace.StatusCodeUnset)
```

Comment:
Shouldn't this be `ptrace.StatusCodeOk`?

Reply:
Fixed bunch of status issues.